get_config.ts

```diff
@@ -1,4 +1,4 @@
-import fs from 'fs';
```

Comment:
why?

Not that I'm opposed, just couldn't see any uses of 'fs' in the diff?

Reply:
ugh this was when my editor went all weird and was throwing an error with the fs types, I was trying everything, reverted now.